scripts/sct_label_vertebrae.py

```diff
@@ -315,9 +315,6 @@ def main(args=None):
             im_label.save(fname_labelz)
         elif fname_initlabel:
             import sct_label_utils
-            # subtract "1" to label value because due to legacy, in this code the disc C2-C3 has value "2", whereas in the
-            # recent version of SCT it is defined as "3". Therefore, when asking the user to define a label, we point to the
-            # new definition of labels (i.e., C2-C3 = 3).
             sct_label_utils.main(['-i', fname_initlabel, '-add', '-1', '-o', fname_labelz])
```

Comment:
to be consistent, if you remove the explanation you also need to remove the code that actually does the "minus one"

Reply:
-I've changed `sct_label_utils` for `Image(fname_initlabel).save(fname_labelz)`. It seems to work fine. 
-Responding to second comment. vertebral_detection in `core.py` uses the precedent convention. Should I shift label value in `core.py`  and see what happens or shift label value in `sct_label_vertebrae`

Reply:
> -I've changed sct_label_utils for Image(fname_initlabel).save(fname_labelz). It seems to work fine.

I don't see your change. Which commit are you referring to? But regardless, my point was that you should not do this "minus one" operation anymore (for the same reason you removed the comment)

Reply:
> -Responding to second comment. vertebral_detection in core.py uses the precedent convention. Should I shift label value in core.py and see what happens or shift label value in sct_label_vertebrae

I don't know which "second comment." you are referring to (there is only one comment in this thread). If you are referring to another post in this PR, a good habit within Github is to add a hyperlink to the post you are referring to.